### DISCUSSION TITLE
ideas that i think are highkry fire for the next release

### DISCUSSION THREAD

**Discussion by notdigidani** [ntsc-rs/ntsc-rs#221] [Ideas]
https://github.com/valadaptive/ntsc-rs/discussions/201
https://github.com/valadaptive/ntsc-rs/discussions/199 ◀ i really like this one right here
https://github.com/valadaptive/ntsc-rs/discussions/209
https://github.com/valadaptive/ntsc-rs/discussions/257 ◀ and this one is lowk the best of em all
https://github.com/valadaptive/ntsc-rs/discussions/268
https://github.com/valadaptive/ntsc-rs/discussions/269
https://github.com/valadaptive/ntsc-rs/discussions/275
https://github.com/valadaptive/ntsc-rs/discussions/288
https://github.com/valadaptive/ntsc-rs/discussions/299
https://github.com/valadaptive/ntsc-rs/discussions/301
https://github.com/valadaptive/ntsc-rs/discussions/302
https://github.com/valadaptive/ntsc-rs/discussions/344
https://github.com/valadaptive/ntsc-rs/discussions/329

thanks it thank you ![image](https://github.com/user-attachments/assets/c0d7269c-23ca-43f8-b4d7-e848b7f1ebf4)

**Comment by nateyboi06** [comment]
would love these, especially 209 :p

**Comment by Glebka73a1** [comment]
half of them are mine lol

**Reply by notdigidani** [reply]
u got a creative mind

**Reply by Glebka73a1** [reply]
ty

**Comment by LucianoTheWindowsFan** [comment]
#69 is perhaps my personal favorite.
